test/fake-timers-test.js

```diff
@@ -4103,6 +4103,62 @@ describe("FakeTimers", function () {
         });
     });
 
+    describe("jump", function () {
+        beforeEach(function () {
+            this.clock = FakeTimers.install({ now: 0 });
+        });
+
+        afterEach(function () {
+            this.clock.uninstall();
+        });
+
+        it("ignores timers which wouldn't be run", function () {
+            const stub = sinon.stub();
+            this.clock.setTimeout(stub, 1000);
+
+            this.clock.jump(500);
+
+            assert(stub.notCalled);
+        });
+
+        it("pushes back execution time for skipped timers", function () {
+            const stub = sinon.stub();
+            this.clock.setTimeout(stub, 1000);
+
+            this.clock.jump(2000);
+
+            assert(stub.calledOnce);
```

Comment:
Are you also able to test that `stub` is called when the time is `2000` and not when it is `1000` (as it would have been with `clock.tick(2000)`)?

Reply:
Good idea! I've updated the test.